pkgs/by-name/me/meshtasticd-bin/package.nix

```
@@ -0,0 +1,89 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  dpkg,
+  i2c-tools,
+  libX11,
+  libgpiod_1,
+  libinput,
+  libusb1,
+  libuv,
+  libxkbcommon,
+  udevCheckHook,
+  ulfius,
+  yaml-cpp,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "meshtasticd-bin";
```

```suggestion
  pname = "meshtasticd";
```
Why the bin? Are we arch linux?